backend/DomainModels/Variant.cs

```diff
@@ -1,7 +1,38 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
 namespace backend.DomainModels;
 
 public class Variant
 {
-    public string Id { get; set; }
-    public string? Name { get; set; }
+    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+    public required int Id { get; set; }
```

Comment:
Could we have a name-generated ID here?

A Function based on first and lastname + int number ? (e.g. magnus-gule-001) etc

Reply:
When we browse the db this improves visibility on FK/PK relations. :)